great_expectations/dataset/sqlalchemy_dataset.py

```diff
@@ -145,6 +145,68 @@ def inner_wrapper(self, column, result_format = None, *args, **kwargs):
 
         return inner_wrapper
 
+    @classmethod
```

Comment:
While this makes sense conceptually, I think it would be clearer, given that only two lines are changing, to adapt the existing column_map_expectation to be able to support either inclusion or exclusion of null values as part of the calculation of statistics regarding success.

What if, instead of just returning the condition, we have the column_map_expectation also return a flag include_nulls_in_condition that allows one to specify whether the condition is allowed to include null values (and update the documentation to include that?